src/content/docs/accounts/accounts-billing/new-relic-one-user-management/tutorial-add-new-user-groups-roles-new-relic-one-user-model.mdx

```diff
@@ -104,7 +104,9 @@ To view existing roles: from the [account dropdown](/docs/using-new-relic/welcom
 
 To create a custom role, click **Add new custom role**. Review the list of available capabilities and decide which ones your custom role needs. 
 
-For more information about how roles and capabilities work, see [Capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure#standard-roles). 
+For more information about how roles and capabilities work, see [Capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure#standard-roles). Or check out this NerdByte video.
+
+<Video id="CBa0jWINQiw" type="youtube" />
 
```

Comment:
When I look at this on Gatsby preview https://docswebsitedevelop-nerdbytesaddtodocs.gtsb.io/docs/accounts/accounts-billing/new-relic-one-user-management/tutorial-add-new-user-groups-roles-new-relic-one-user-model/ It does not show running time, which style guide says you should include. https://docs.newrelic.com/docs/style-guide/processes-procedures/embed-images/#add  (It's odd that this is in the "embed images" not "embed videos" doc.) People like me like to know what I'm getting myself into before committing :-) and it's easier to decide if I know how long it will take.

Reply:
Example wording in https://docs.newrelic.com/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/ : "For a lesson in how to use NerdGraph, watch this 7-minute video:"